### PR TITLE
Ret 2183

### PR DESCRIPTION
**Pull request by robin-ieong**
https://tools.hmcts.net/jira/browse/RET-2182
https://tools.hmcts.net/jira/browse/RET-2183
https://tools.hmcts.net/jira/browse/RET-2192

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x ] No
```

### CODE REVIEW

File: definitions/json/CaseField.json [hunk 1]
```diff
@@ -2981,6 +2981,102 @@
...
+    "CaseTypeID": "ET_EnglandWales",
+    "ID": "referralsLabel",
+    "Label": "**Referrals**",
+    "FieldType": "Label",
+    "SecurityClassification": "Public"
+  },
+  {
+    "CaseTypeID": "ET_EnglandWales",
+    "ID": "referralLinks",
+    "Label": "<a href=\"/cases/case-details/${[CASE_REFERENCE]}/trigger/createReferral/createReferral1\">Send a new referral</a> <br> <a href=\"/cases/case-details/${[CASE_REFERENCE]}#Documents\" target=\"_blank\">Reply to a referral</a> <br> <a href=\"/cases/case-details/${[CASE_REFERENCE]}#Documents\" target=\"_blank\">Close a referral</a>",
```
Comment (Brogie) [inline definitions/json/CaseField.json:2994]: test this in demo apparently there is issues with this method of linking

**Review by Brogie — APPROVED** [review]

File: definitions/json/CaseEventToFields.json [hunk 2]
```diff
@@ -2745,6 +2745,16 @@
     "PageFieldDisplayOrder": 6,
     "ShowSummaryChangeOption": "Y"
   },
+  {
+    "CaseTypeID": "ET_EnglandWales",
+    "CaseEventID": "et1Vetting",
+    "CaseFieldID": "trackType",
+    "FieldShowCondition": "et1JurisdictionCodeGeneralNotes=\"dummy\"",
```
Comment (hsjhita) [inline definitions/json/CaseEventToFields.json:2752]: Not the best of fieldShow conditions - what if someone actually typed in dummy on the general notes box? This field would then show

File: definitions/json/CaseField.json [hunk 3]
```diff
@@ -2981,6 +2981,116 @@
...
+    "CaseTypeID": "ET_EnglandWales",
+    "ID": "referralsLabel",
+    "Label": "**Referrals**",
+    "FieldType": "Label",
+    "SecurityClassification": "Public"
+  },
+  {
+    "CaseTypeID": "ET_EnglandWales",
+    "ID": "referralLinks",
+    "Label": "<a href=\"/cases/case-details/${[CASE_REFERENCE]}/trigger/createReferral/createReferral1\">Send a new referral</a> <br> <a href=\"/cases/case-details/${[CASE_REFERENCE]}/trigger/replyToReferral/replyToReferral1\">Reply to a referral</a> <br> <a href=\"/cases/case-details/${[CASE_REFERENCE]}#Documents\" target=\"_blank\">Close a referral</a>",    "FieldType": "Label",
```
Comment (hsjhita) [inline definitions/json/CaseField.json:3002]: FieldType on new line

File: definitions/json/CaseTypeTab.json [hunk 4]
```diff
@@ -1003,27 +1029,27 @@
...
     "DisplayContextParameter": "#TABLE(bfDate)"
   },
   {
     "CaseTypeID": "ET_EnglandWales",
     "Channel": "CaseWorker",
-    "TabID": "CaseHistory",
-    "TabLabel": "History",
-    "TabDisplayOrder": 14,
-    "CaseFieldID": "caseHistory",
+    "TabID": "CaseDocuments",
```
Comment (hsjhita) [inline definitions/json/CaseTypeTab.json:1040]: History should be tab 15, Document should be tab 16. This was a change made to resolve an ExUI bug

**Review by JackReevies — APPROVED** [review]

File: definitions/json/CaseEventToFields.json [hunk 5]
```diff
@@ -5332,6 +5342,122 @@
...
+    "PageColumnNumber": 1
+  },
+  {
+    "CaseTypeID": "ET_EnglandWales",
+    "CaseEventID": "replyToReferral",
+    "CaseFieldID": "selectReferralToReply",
+    "DisplayContext": "MANDATORY",
+    "PageID": 1,
+    "PageDisplayOrder": 1,
+    "PageFieldDisplayOrder": 1,
```
Comment (JackReevies) [inline definitions/json/CaseEventToFields.json:5458]: Should this be 11?

File: definitions/json/CaseField.json [hunk 6]
```diff
@@ -2988,6 +2988,117 @@
...
+    "Label": "${referralHearingDetails}",
+    "FieldType": "Label",
+    "SecurityClassification": "Public"
+  },
+  {
+    "CaseTypeID": "ET_EnglandWales",
+    "ID": "referCaseTo",
+    "Label": "Who are you referring this case to?",
+    "FieldType": "FixedRadioList",
+    "FieldTypeParameter": "fl_ReferCaseTo",
```
Comment (JackReevies) [inline definitions/json/CaseField.json:3032]: frl_ReferCaseTo? since its a Fixed Radio List not a Fixed List

**Review by JackReevies — CHANGES_REQUESTED** [review]
see comments

File: definitions/json/CaseEventToFields.json [hunk 7]
```diff
@@ -2745,11 +2745,21 @@
...
+    "PageID": 5,
+    "PageDisplayOrder": 5,
+    "PageFieldDisplayOrder": 7
+  },
   {
     "CaseTypeID": "ET_EnglandWales",
     "CaseEventID": "et1Vetting",
     "CaseFieldID": "trackAllocation",
-    "FieldShowCondition": "et1SubstantiveDefectsGeneralNotes=\"dummy\"",
+    "FieldShowCondition": "et1SubstantiveDefectsGeneralNotes=\"dummy dummy dummy dum dummy\"",
```
Comment (hsjhita) [inline definitions/json/CaseEventToFields.json:2762]: What I meant with my previous comment on this was not to set a text field as the show condition. The user can still type in dummy dummy dummy dum dummy and this field would appear. Maybe set the field for the condition as a YesNo field.

File: definitions/json/CaseEventToFields.json (hunk 5 above)
Comment (hsjhita) [reply definitions/json/CaseEventToFields.json:5458]: Different CaseEventID

**Review by hsjhita — APPROVED** [review]

**Pull request merged**